spec/ruby/core/fiber/storage_spec.rb

```diff
@@ -0,0 +1,111 @@
+require_relative '../../spec_helper'
+
+require 'fiber'
+
+describe "Fiber.new(storage:)" do
+  ruby_version_is "3.2" do
```

Comment:
Minor: the `ruby_version_is` should be the outermost block, and then contain everything else.

Reply:
Ah, thanks I didn't know that was possible. I'll fix it.